linux.md

```diff
@@ -199,6 +199,7 @@ Make sure that you're running the 2nd-newest OS version or the newest version -
     sudo apt update
     sudo apt install postgresql-16
     echo "export PSQL_PAGER=\"less --chop-long-lines --header 1\"" >> ~/`[[ $SHELL == *"zsh" ]] && echo '.zshenv' || echo '.bashrc'`
+    sed -i "/^[[:space:]]*lc_messages[[:space:]]*=/ s/^#*.*$/lc_messages = 'en_US.UTF-8'/" "$(sudo -u postgres psql --tuples-only --pset format=unaligned --command 'SHOW config_file;')"
```

Comment:
This is setting the `lc_messages` to utf-8 in Linux using sed. It gets the `postgresql.conf` file path dynamically and passes that path to sed to make the update

Reply:
hm, seems pretty nested and it may be hard to see what failed - probably should be 2 lines:

1. one first line to set a temporary shell variable (check other guides for temporary variable naming) to the `config_file` value
2. 2nd line to use this variable with `sed`

Reply:
good start, I guess this should be on 2 lines though

- seems pretty nested and it may be hard to see what failed
- also, it makes it harder to update the Linux guide, if we make changes to the `sed` command
  - let's keep the `sed` command the same between all 3 guides

so the 2 lines could be:

1. one first line to set a shell variable to `PGDATA` (temporary variable, no need to set it in `.bashrc` or `.zshenv`) to the `config_file` value
2. 2nd line to use this variable with `sed`

Reply:
done in 07ae0993021aafbe3dd5799f3cebaddf2da9ec29